assembly/integer/u128.ts

```diff
@@ -463,11 +462,11 @@ export class u128 {
   }
 
   @inline @operator('%')
-  static rem(a: u128, b: u128): u128 {
```

Comment:
I think you should cast result to `u128` anyway. Like: `return u128.from(__divmod_rem)` or `return new u128(__divmod_rem)`